src/main/scala/viper/server/vsi/Cache.scala

```diff
@@ -46,10 +48,16 @@ abstract class Cache {
     * The inner map is referred to as the fileCache. As the name indicates, it stores, for each
     * file, a number of hashes and corresponding cache entries.
     */
-  protected val _cache: MutableMap[String, FileCash] = MutableMap()
-  type FileCash = MutableMap[String, List[CacheEntry]]
+  type FileKey = String
+  type MethodHash = String
```

Comment:
```suggestion
  type MemberHash = String
```
Aren't functions and predicates cached as well?